### PR TITLE
Disable cache versioning

### PR DESCRIPTION
**Pull request by krauselukas**
Since we lately experienced some issues related to the newly
introduced cache versioning, we should disable it for now and
first adjust/check our codebase accordingly.

### CODE REVIEW

**Comment by hennevogel** [comment]
So this config option is going away later in Rails?

**Comment by krauselukas** [comment]
> So this config option is going away later in Rails?

@hennevogel I couldn't find anything mentioning that this will be dropped, I checked the 6.1 Milestones etc.

File: src/api/config/application.rb [hunk 1]
```diff
@@ -87,6 +87,12 @@ class Application < Rails::Application
 
     config.action_controller.perform_caching = true
 
+    # FIXME: Check all the places in the codebase where we cache content
```
Comment (krauselukas) [inline src/api/config/application.rb:90]: @hennevogel so maybe not marking as fixme?

Comment (hennevogel) [reply src/api/config/application.rb:90]: yeah just disable it! :)

**Pull request merged**